hack/chart/appscode-searchlight/templates/_helpers.tpl

```diff
@@ -12,5 +12,5 @@ We truncate at 24 chars because some Kubernetes name fields are limited to this
 */}}
 {{- define "fullname" -}}
 {{- $name := default .Chart.Name .Values.nameOverride -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 24 -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 40 -}}
```

Comment:
NAME                                                              READY     STATUS     RESTARTS   AGE
appscode-icinga-halting-grizzly-appscode-searchlight-83470b4rbj   2/2       Running    0          20s
We get name like this if we use 40. We did it to get the full name (appscode-icinga-halting-grizzly-appscode-searchlight) if we use 24 we get (appscode-icinga-whimsical-ferrit-appscod-3115551314-sf0ht) The name is cut down.